locales/ja/ja.json

```diff
@@ -694,17 +694,17 @@
     "Venezuela": "ベネズエラ",
     "Venezuela, Bolivarian Republic of": "ベネズエラ・ボリバル共和国",
     "Verify Email Address": "メールアドレスを確認してください",
-    "Verify Your Email Address": "Verify Your Email Address",
+    "Verify Your Email Address": "メールアドレスの確認",
     "Viet Nam": "ベトナム",
     "View": "ビュー",
-    "View Receipt": "View Receipt",
+    "View Receipt": "領収書を見る",
     "Virgin Islands, British": "英領ヴァージン諸島",
     "Virgin Islands, U.S.": "米領ヴァージン諸島",
     "Wallis And Futuna": "ウォリス・フツナ",
     "Wallis and Futuna": "ウォリス・フツナ",
-    "We are processing your subscription. Once the subscription has successfully processed, this page will update automatically. Typically, this process should only take a few seconds.": "We are processing your subscription. Once the subscription has successfully processed, this page will update automatically. Typically, this process should only take a few seconds.",
+    "We are processing your subscription. Once the subscription has successfully processed, this page will update automatically. Typically, this process should only take a few seconds.": "私たちはあなたの購読を処理しています。 サブスクリプションが正常に処理されると、このページは自動的に更新されます。 通常、このプロセスには数秒しかかかりません。",
     "We are unable to process your payment. Please contact customer support.": "決済が完了できませんでした。カスタマーサポートへお問い合わせください。",
-    "We have emailed your password reset link!": "We have emailed your password reset link!",
+    "We have emailed your password reset link!": "私たちはあなたのパスワードリセットリンクを電子メー",
```

Comment:
Is it correct that there is no exclamation point at the end of a sentence? It is present in other proposals. For example, here:

```
"Failed to load :resource!": "読み込みに失敗しました ：resource！",
```